doc/getting-started/roscube/roscube-gsg.rst

```diff
@@ -433,10 +454,15 @@ the User VM.
      wget https://raw.githubusercontent.com/Adlink-ROS/ROScube_ACRN_guide/v2.1/scripts/launch_ubuntu_uos.sh
```

Comment:
Nitpick: it would make sense to create a new v2.5 tag in that repo (even though the content may be the same as in v2.1)

Reply:
I submitted an issue to the ROScube doc repo: https://github.com/Adlink-ROS/ROScube_ACRN_guide/issues/2